jobs/pull-kubernetes-federation-e2e-gce-gci.sh

```diff
@@ -47,7 +47,7 @@ export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
 # Push federation images to GCS.
-./build-tools/push-federation-images.sh
+./build/push-federation-images.sh
```

Comment:
I think you can actually just remove these - hack/jenkins/build.sh will do this for you now.

feel free to tag on another commit to this PR removing.